graphql/main.go

```diff
@@ -26,18 +32,42 @@ func main() {
 
 	app := fiber.New()
 
-	app.Post("/", func(ctx *fiber.Ctx) error {
-		query := string(ctx.Body())
-		params := graphql.Params{Schema: schema, RequestString: query}
-		r := graphql.Do(params)
-		if len(r.Errors) > 0 {
-			ctx.SendString("failed to execute graphql operation")
+	// curl 'http://localhost:9090/?query=query%7Bhello%7D'
+	app.Get("/", func(ctx *fiber.Ctx) error {
+		var input Input
+		if err := ctx.QueryParser(&input); err != nil {
+			return ctx.
+				Status(http.StatusInternalServerError).
```

Comment:
Uae `fiber.StatusInternalServerError` that way we dont have to import net/http

Reply:
Oh, you're right => Done